apps/odyssey-stats/src/lib/set-locale.ts

```diff
@@ -46,25 +26,15 @@ const convertPhpToMomentFormat = ( phpFormat: string ): string => {
  */
 const fixLongDateFormatForEn = ( localeSlug: string ) => {
 	if ( localeSlug === SIMPLE_SITE_DEFAULT_LOCALE ) {
-		const keysReplacedByWordPressCore: LongDateFormatKey[] = [
-			'LTS',
-			'LT',
-			'L',
-			'LL',
-			'LLL',
-			'LLLL',
-		];
-		const currentLocaleData = moment.localeData();
-		const momentLongDateFormat: Partial< { [ key in LongDateFormatKey ]: string } > = {};
-		keysReplacedByWordPressCore.forEach( ( key ) => {
-			if ( currentLocaleData.longDateFormat( key ) ) {
-				momentLongDateFormat[ key ] = convertPhpToMomentFormat(
-					currentLocaleData.longDateFormat( key )
-				);
-			}
-		} );
 		moment.updateLocale( localeSlug, {
-			longDateFormat: momentLongDateFormat as { [ key in LongDateFormatKey ]: string },
+			longDateFormat: {
```

Comment:
It will potentially discard any modification by WP here? Would it be okay? It'll be great if you could share you insights here @candy02058912. Thanks 🙇